share/instruments/instruments.xml

```diff
@@ -7867,346 +7844,7 @@
><descr>set stop Sifflet 1</descr><controller ctrl="98" value="96"/><controller ctrl="98" value="8"/></MidiAction><MidiAction name="cymb"><descr>set stop Cymbel VI</descr><controller ctrl="98" value="96"/><controller ctrl="98" value="9"/></MidiAction><MidiAction name="oboe"><descr>set stop Oboe</descr><controller ctrl="98" value="96"/><controller ctrl="98" value="10"/></MidiAction><MidiAction name="TR"><descr>set stop Tremulant</descr><controller ctrl="98" value="96"/><controller ctrl="98" value="11"/></MidiAction></Channel><Channel name="P"><descr>Division P (pedal)</descr><synti>Aeolus</synti><program value="0"/><MidiAction name="Reset"><descr>Reset all stops</descr><controller ctrl="98" value="67"/></MidiAction><MidiAction name="subb16"><descr>set stop Subbass 16</descr><controller ctrl="98" value="99"/><controller ctrl="98" value="0"/></MidiAction><MidiAction name="prin16"><descr>set stop Principal 16</descr><controller ctrl="98" value="99"/><controller ctrl="98" value="1"/></MidiAction><MidiAction name="prin8"><descr>set stop Principal 8</descr><controller ctrl="98" value="99"/><controller ctrl="98" value="2"/></MidiAction><MidiAction name="prin4"><descr>set stop Principal 4</descr><controller ctrl="98" value="99"/><controller ctrl="98" value="3"/></MidiAction><MidiAction name="oct2"><descr>set stop Octave 2</descr><controller ctrl="98" value="99"/><controller ctrl="98" value="4"/></MidiAction><MidiAction name="oct1"><descr>set stop Octave 1</descr><controller ctrl="98" value="99"/><controller ctrl="98" value="5"/></MidiAction><MidiAction name="qu513"><descr>set stop Quint 5 1/3</descr><controller ctrl="98" value="99"/><controller ctrl="98" value="6"/></MidiAction><MidiAction name="qu223"><descr>set stop Quint 2 2/3</descr><controller ctrl="98" value="99"/><controller ctrl="98" value="7"/></MidiAction><MidiAction name="mixt"><descr>set stop Mixture</descr><controller ctrl="98" value="99"/><controller ctrl="98" value="8"/></MidiAction><MidiAction name="bass18"><descr>set stop Fagott 16</descr><controller ctrl="98" value="99"/><controller ctrl="98" value="9"/></MidiAction><MidiAction name="trom16"><descr>set stop Trombone 16</descr><controller ctrl="98" value="99"/><controller ctrl="98" value="10"/></MidiAction><MidiAction name="bomb32"><descr>set stop Bombarde 32</descr><controller ctrl="98" value="99"/><controller ctrl="98" value="11"/></MidiAction><MidiAction name="trum8"><descr>set stop Trumpet</descr><controller ctrl="98" value="99"/><controller ctrl="98" value="12"/></MidiAction><MidiAction name="CP+1"><descr>set stop P+I</descr><controller ctrl="98" value="99"/><controller ctrl="98" value="13"/></MidiAction><MidiAction name="CP+2"><descr>set stop P+II</descr><controller ctrl="98" value="99"/><controller ctrl="98" value="14"/></MidiAction><MidiAction name="CP+3"><descr>set stop P+III</descr><controller ctrl="98" value="99"/><controller ctrl="98" value="15"/></MidiAction></Channel><genre>common</genre><genre>orchestra</genre></Instrument>
```

Comment:
Something went wrong here

Reply:
It i just commented out.

Reply:
It was commended out before but now it turned into a one-liner

Reply:
Let's delete it.

Reply:
along with all the other disabled reminders of Aeolus?